src/components/accessible-autocomplete/README.md

```diff
@@ -0,0 +1,64 @@
+# Accessible Autocomplete
+
+This macro takes an `<select>` and uses progressive enhancement, provided by the govuk accessible-autocomplete enhanceSelectElement function.
+It also supports a `<label>` with hint and error message support for the `<select>`
+
+The enhanceSelectElement will transform the `<select>` into an `<input>` with full accessibility support.
+
+## Parameters
+
+```
+select
+id
+hint
+errorMessage
+describedBy
+```
+
+## Parameters for the select input
+
+Look [here](https://www.npmjs.com/package/accessible-autocomplete) for more information on the attributes for the accessible autocomplete component.
+
+## id
+Should be the same as the id for this component
+
+## data-default-value
+Type: Boolean
+
+If this is set to true, all values are shown when the user clicks the input. This is similar to a default dropdown, so the autocomplete is rendered with a dropdown arrow to convey this behaviour
```

Comment:
This looks like the type/description for `data-show-all-values` - is this right?

Reply:
Thanks for spotting that, i have updated it to the correct description.